packages/core-typings/src/voip/SignallingSocketEvents.ts

```diff
@@ -0,0 +1,8 @@
+export type SignallinSocketEvents = {
```

Comment:
small typo (missing `g`) .. also suggested to use single `l` per Kevin's suggestion on the translation.. would be good to have the file name changed as well.
```suggestion
export type SignalingSocketEvents = {
```

Reply:
Fixed. Renamed the file.